rapid_response_xblock/utils.py

```diff
@@ -0,0 +1,31 @@
+"""Utils methods for instructor dashboard"""
+
+from rapid_response_xblock.models import RapidResponseRun, RapidResponseSubmission
+
+
+def get_problem_runs(course_key):
+    """Util method to return problem runs corresponding to given course key"""
+    problem_runs = RapidResponseRun.objects.filter(course_key=course_key).values('id', 'created', 'problem_usage_key')
+    return problem_runs
+
+
+def get_rapid_response_report_data(run_id):
```

Comment:
@gsidebo Do we need to add tests to these methods? I have added test for `get_problem_runs` but I am not sure how I can add test for this method. How I can create submission against problem ?

Reply:
One other thing: I would call this `get_run_submission_data`. This method returns submission data associated with some rapid response run. The current client is making a report out of that data, but that is a concern of the client